Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.6
+FROM golang:1.9.1-alpine3.6
```

Comment:
I noticed that this script install go 1.8.3 (`(16/18) Installing go (1.8.3-r0)`)

In addition, isn't changing from 'alpine:3.6'  to 'golang:1.9.1-alpine3.6' we don't have to install go anymore ?

Reply:
we don't @vadmeste . let me fix that.